### PR TITLE
73 move location filtering

### LINKED ISSUE #73: move location filtering to model data prep
We currently filter locations in the very first step in the workflow:  [query_meta_keys.py](https://github.com/OxfordDemSci/UkraineNowPop/blob/dev/src/1_pop_data/query_meta_keys.py#L30).

If we remove locations there, then the covariates will not be generated for those locations (see work on issue #56). To avoid the need to re-run the entire covariates processing pipeline when locations are added or removed for a particular model, we need to move location filtering to a later step in the pipeline when we are preparing the model data object `md`.

### PR DESCRIPTION
**Pull request by doug-leasure**
@edarin are you happy with these changes?

1. Removed location filtering from `./src/1_pop_data/query_meta_keys.py`
2. Added location filtering to `./src/2_model/models/1_base_model_config.R`
3. A few minor housekeeping changes to revise little glitches I found re-running the pipeline

Note: I did not copy the new `meta_keys.csv` to the network drive.

### CODE REVIEW

**Review by edarin — COMMENTED** [review]
I thought we said that meta_keys.py should no be 00 as meta_keys.csv will be given as an input file for any other people wanting to replicate this work and that have not sql access to the DB?

**Comment by doug-leasure** [comment]
> I thought we said that meta_keys.py should no be 00 as meta_keys.csv will be given as an input file for any other people wanting to replicate this work and that have not sql access to the DB?

We either need to save the output into the repository so that it is available or have it be a numbered script. Otherwise, it is unclear where to get the data. I'm happy either way.

**Comment by doug-leasure** [comment]
@edarin , moved meta_keys.csv and cod-ps to `./data/`. Have a look at the `border_crossings.R` script to see if you want to number it differently to be consistent with your covariates pipeline.

File: src/1_pop_data/10_create_master_index.R
Comment (edarin) [inline src/1_pop_data/10_create_master_index.R:1]: I am changing my mind. I do find a bit dangerous to have in data and not in the shared input folder. It is important that we are sourcing the same data input. Typically for meta_keys for example. So I would suggest either to:
- rever to /input in the shared drive
- having data as apart of github and thus keeping very small. I'm in favour of this one but then /data shout be out of the git ignore. Would that fit your pipeline?

Comment (doug-leasure) [reply src/1_pop_data/10_create_master_index.R:1]: I will take ./data out of the gitignore. I previously also wanted to do that when I saw that GISRede had saved data there.

**Review by edarin — APPROVED** [review]
I have added cod-ab and pop_baseline that are required in other scripts

**Pull request merged**